src/frontend/components/avatar.js

```diff
@@ -14,24 +22,64 @@ export class Avatar extends LitElement {
       align-items: center;
       overflow: hidden;
     }
+  `;
 
-    .avatar img {
-      width: 100%;
-      height: 100%;
-      object-fit: cover;
-      object-position: center;
-    }
-    `
+  constructor() {
+    super();
+    this.initials = "";
+    this.avatar = "";
+    this.name = "";
+    this.color = "";
+    this.id_user = "";
+    this.bAvatar = true;
+  }
 
-    
-    render() {
-        return html`
-            <div class="avatar">
-                <img src=${Immagine} />
-            </div>
-        `
+  render() {
+    if (this.avatar == "#") {
```

Comment:
estrarre le logiche di definizione dell'immagine di default dalla funzione render in una funzione apposita